.eslintignore

```diff
@@ -0,0 +1 @@
+public/
```

Comment:
Per the module template, this should be in an `ignorePatterns` array in `.eslintrc.js`.

Reply:
i tried `ignorePatterns` with:

```
public/**/*.js
```

still included it, not sure the magical incantation